great-idea/css/index.css

```diff
@@ -63,4 +63,112 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+body {
+	margin: 60px 100px;
+  }
+  
+  header nav a {
+	color: gray;
+	margin: 35px 3%;
+	text-decoration: none;
+  }
+  
+  header nav {
+	width: 80%;
+  }
+  
+  header {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	margin-bottom: 80px
+  }
+  
+  h1 {
+	font-size: 80px;
+	padding: 20px;
+  }
+  
+  article {
+	text-align: center;
+	display: flex;
+	align-content: space-between;
+  }
+  
+  button:hover {
+	background-color: black;
+	color: white;
+  }
+  
+  button {
+	background-color: white;
+	border: 1px solid black;
+	padding: 7px 45px;
+	font-size: 16px;
+  }
+  
+  .top-content {
+	width: 49%;
+	height: 450px;
+  }
+  
+  .circle-pic {
+	width: 49%;
+  }
+  
+  .content {
+	border-style: solid none solid none;
+	border-width: 1px;
+  }
+  
+  .center-content-header{
+	display: flex;
+	
+  }
+  
+  section h2 {
```

Comment:
Be cautious and make sure you plan on using the HTML tag selectors like this. Remember this will be applied to ALL h2 tags nested inside of section tags(with low specificity). Just know that could be the cause of unwanted behavior when it occurs.